lua/lint/linters/sqlfluff.lua

```diff
@@ -35,8 +35,8 @@ return {
         for _, violation in ipairs(i_filepath.violations) do
           table.insert(diagnostics, {
             source = 'sqlfluff',
-            lnum = violation.line_no - 1,
-            col = violation.line_pos - 1,
+            lnum = violation.start_line_no - 1,
+            col = violation.start_line_pos - 1,
```

Comment:
```suggestion
            lnum = (violation.line_no or violation.start_line_no) - 1,
            col = (violation.line_pos or violation.start_line_pos) - 1,
```

To stay compatible with older versions